packages/opentelemetry-metrics/src/Meter.ts

```diff
@@ -130,7 +137,35 @@ export class Meter implements types.Meter {
   getMetrics(): ReadableMetric[] {
     return Array.from(this._metrics.values())
       .map(metric => metric.get())
-      .filter(metric => !!metric);
+      .filter(notNull);
+  }
+
+  /**
+   * Add an exporter to the list of registered exporters
+   *
+   * @param exporter exporter to add to the list of registered exporters
+   */
+  addExporter(exporter: MetricExporter) {
+    this._exporters.push(exporter);
+  }
+
+  /**
+   * Send a single metric by name to all registered exporters
+   */
+  private _exportOneMetric(name: string) {
+    const metric = this._metrics.get(name);
```

Comment:
What do you think about removing nested if statement with somthing like this?
```typescript
const metric = this._metrics.get(name);
if (!metric) return;
const readableMetric = metric.get();
if (!readableMetric) return;

for (const exporter of this._exporters) {
  exporter.export([readableMetric], result => {
  if (result !== ExportResult.SUCCESS) {
    this._logger.error(`Failed to export ${name}`);
  }
});
```

Reply:
For a practical reason or just style? I don't mind I'm just curious if there's something I'm not thinking of.

Reply:
Just a style suggestion, feel free to ignore if you disagree.

Reply:
done